editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/globalconfiguration/beans/provider/GlobalBeanContributor.java

```diff
@@ -74,7 +75,18 @@ public void onGlobalElementDeleted(AbstractCamelModelElement camelModelElement)
 
 	@Override
 	public boolean canHandle(AbstractCamelModelElement camelModelElementToHandle) {
-		return CamelUtils.getTranslatedNodeName(camelModelElementToHandle.getXmlNode()).equalsIgnoreCase(CamelFile.BEAN_NODE);
+		// to avoid clash with SAP extension, needs to NOT handle a bean with the class 
+		// org.fusesource.camel.component.sap.SapConnectionConfiguration
+		boolean isBeanElement = 
+				CamelUtils.getTranslatedNodeName(camelModelElementToHandle.getXmlNode()).equalsIgnoreCase(CamelFile.BEAN_NODE);
+		boolean isNotSAPClass = true;
+		if (isBeanElement) {
+			Object classParm = camelModelElementToHandle.getParameter(CamelBean.PROP_CLASS);
+			if (classParm != null && classParm instanceof String) {
```

Comment:
I think that you can simply call "org.fusesource.camel.component.sap.SapConnectionConfiguration".equals(classParm)